core/lib/dal/src/witness_generator_dal.rs

```diff
@@ -728,7 +728,7 @@ impl WitnessGeneratorDal<'_, '_> {
         {
             sqlx::query!(
                 "INSERT INTO witness_inputs(l1_batch_number, merkle_tree_paths, merkel_tree_paths_blob_url, status, protocol_version, created_at, updated_at) \
-                 VALUES ($1, $2, $3, 'queued', $4, now(), now())
+                 VALUES ($1, $2, $3, 'waiting_for_artifacts', $4, now(), now()) \
```

Comment:
Do I get it right that this state is rather temporary and will be removed once we fully switch to using BWIP?